packages/react-composites/src/composites/CallComposite/utils/Utils.ts

```diff
@@ -96,9 +96,21 @@ export const getCallCompositePage = (
   // Must check for ongoing call *before* looking at any previous calls.
   // If the composite completes one call and joins another, the previous calls
   // will be populated, but not relevant for determining the page.
+  call?.remoteParticipants;
   if (call) {
+    // For PSTN/ACS Calls, the composite should display the call screen immediately.
```

Comment:
Please add a comment *why* we should skip the call screen. It is not obvious to me.

Is it because we don't want to transition the UX through two transitory states: Connecting screen -> Ringing tile? Or is there another reason?